pac4j-core/src/main/java/org/pac4j/core/client/DirectClientV2.java

```diff
@@ -38,7 +39,14 @@ protected C retrieveCredentials(final WebContext context) throws HttpAction {
             this.authenticator.validate(credentials, context);
             return credentials;
         } catch (CredentialsException e) {
-            logger.error("Failed to retrieve or validate credentials", e);
+            if (logger.isDebugEnabled()) {
```

Comment:
Cool, but you're always logging at `loger.info`? Shouldnt one of these calls be `logger.debug`?